x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StartTrainedModelDeploymentAction.java

```diff
@@ -734,7 +734,7 @@ public static long estimateMemoryUsageBytes(
         }
     }
 
-    private static boolean isElserV1Model(String modelId) {
-        return modelId.startsWith(".elser_model_1");
+    private static boolean isElserV1Or2Model(String modelId) {
+        return modelId.startsWith(".elser_model_1") || modelId.startsWith(".elser_model_2");
```

Comment:
Will the elser v2 model have the platform version appended to it?

Reply:
Yes, there will be two variants of ELSER v2: `.elser_model_2` and `.elser_model_2_linux-x86_64`. There are also `.elser_model_2_SNAPSHOT` and `.elser_model_2_linux-x86_64_SNAPSHOT` which will be available even before feature freeze. But all four of them start with `.elser_model_2` so we're assuming they all have the same memory requirement.

Reply:
Oh I missed the `startsWith` so this works fine 🤦‍♂️